octobot_evaluators/evaluators/abstract_evaluator.py

```diff
@@ -545,6 +546,27 @@ def ensure_eval_note_is_not_expired(self) -> None:
                 self.eval_note_time_to_live = None
                 self.eval_note_changed_time = None
 
+    def _log_on_invalid_eval_not_time(self, exchange_name, matrix_id, symbol, eval_time, time_frame) -> None:
+        if not time_frame:
+            return
+        current_time = self._get_exchange_current_time(exchange_name, matrix_id)
+        if not matrix.is_evaluation_valid_in_time(current_time, eval_time, commons_enums.TimeFrames(time_frame)):
+            self.logger.warning(
+                f"Performed {symbol} {time_frame} evaluation on outdated data: {eval_time=}, {current_time=}. "
+                f"Up-to-date {symbol} price data on {time_frame} might not yet be available on {exchange_name}."
```

Comment:
without this, the strategy is just not called without any info, making the situation hard to understand